draft-jones-perc-dtls-tunnel/draft-jones-perc-dtls-tunnel.md

```diff
@@ -169,7 +178,7 @@ The key distributor is a logical function that might might be co-resident
 with a key management server operated by an enterprise, reside in one of
 the endpoints participating in the conference, or elsewhere that is
 trusted with E2E keying material.  This document does not preclude any
-location, only requiring that the key distributor not allow the media
+location, only requiring that the key distributor does not allow the media
```

Comment:
We should not have the word "does" here.  But, I can see how this sentence is confusing.  It's also redundant.  We've already stated that the key server can be anywhere that is secure, and its location has nothing to do with the MD getting the keying material. Shall we remove it?